data/libraries.json

```diff
@@ -1,13 +1,13 @@
 [
     {
-        "last_renewed": null,
+        "last_renewed": "2017-03-29T00:32:11",
         "name": "agsXMPP SDK",
         "platforms": [
             ".net",
             "C#",
             "Mono"
         ],
-        "url": "http://ag-software.net"
+        "url": "https://www.ag-software.net"
```

Comment:
There appears to be a more specific URL for the agsXMPP SDK project: https://www.ag-software.net/agsxmpp-sdk/